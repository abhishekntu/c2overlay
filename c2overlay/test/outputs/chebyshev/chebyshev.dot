digraph graphname {
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="N2_mul_N2_N2"];
N3 [color="black", label="N3_mul_N3_N3"];
N4 [color="black", label="N4_mul_Imm_16_N4_N4"];
N5 [color="black", label="N5N7_mul_N5_sub_Imm_20_N7_N5"];
N6 [color="black", label="N6N8_mul_N6_add_Imm_5_N8_N6"];
N7 [color="black", label="O0_N9_N7"];
N1 -> N5;
N1 -> N6;
N1 -> N2;
N1 -> N3;
N1 -> N4;
N2 -> N7;
N3 -> N6;
N4 -> N5;
N5 -> N3;
N6 -> N2;
}
