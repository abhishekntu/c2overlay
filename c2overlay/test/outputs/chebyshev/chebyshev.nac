procedure foo(in s32 x,out s32 D_2404)
{
  localvar s32 D_2405;
  localvar s32 D_2406;
  localvar s32 D_2407;
  localvar s32 D_2408;
  localvar s32 D_2409;
  localvar s32 temp;
L0005:
  temp <= mul x,16;
  D_2405 <= mul temp,x;
  D_2406 <= sub D_2405,20;
  D_2407 <= mul D_2406,x;
  D_2408 <= mul D_2407,x;
  D_2409 <= add D_2408,5;
  D_2404 <= mul D_2409,x;
}
