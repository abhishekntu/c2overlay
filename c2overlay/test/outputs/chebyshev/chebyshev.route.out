Array size: 3 x 3 logic blocks.

Routing:

Net 0 (N1)

SOURCE (3,0)  Pad: 1  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 3  
  IPIN (3,1)  Pin: 2  
  SINK (3,1)  Class: 0  
 CHANX (3,0)  Track: 3  
 CHANX (2,0)  Track: 3  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  
 CHANX (3,0)  Track: 3  
 CHANY (2,1)  Track: 2  
 CHANY (2,2)  Track: 2  
  IPIN (3,2)  Pin: 3  
  SINK (3,2)  Class: 0  
 CHANY (2,2)  Track: 2  
  IPIN (2,2)  Pin: 1  
  SINK (2,2)  Class: 0  
 CHANX (2,0)  Track: 3  
 CHANX (1,0)  Track: 3  
  IPIN (1,1)  Pin: 2  
  SINK (1,1)  Class: 0  


Net 1 (N7)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 6  
 CHANX (1,0)  Track: 2  
  IPIN (1,0)  Pad: 0  
  SINK (1,0)  Pad: 0  


Net 2 (N3)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 7  
 CHANY (2,1)  Track: 3  
  IPIN (2,1)  Pin: 1  
  SINK (2,1)  Class: 0  


Net 3 (N4)

SOURCE (2,2)  Class: 1  
  OPIN (2,2)  Pin: 5  
 CHANY (2,2)  Track: 1  
 CHANX (3,1)  Track: 0  
  IPIN (3,2)  Pin: 2  
  SINK (3,2)  Class: 0  


Net 4 (N5)

SOURCE (3,2)  Class: 1  
  OPIN (3,2)  Pin: 6  
 CHANX (3,1)  Track: 3  
  IPIN (3,1)  Pin: 0  
  SINK (3,1)  Class: 0  


Net 5 (N6)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 7  
 CHANY (1,1)  Track: 3  
  IPIN (1,1)  Pin: 1  
  SINK (1,1)  Class: 0  
